lib/pure/encodings.nim

```diff
@@ -221,14 +251,14 @@ when defined(windows):
     proc getCPInfo(codePage: CodePage, lpCPInfo: var CpInfo): int32 {.
       stdcall, importc: "GetCPInfo", dynlib: "kernel32".}
 
-  proc nameToCodePage*(name: string): CodePage =
+  proc nameToCodePage(name: string): CodePage =
     var nameAsInt: int
     if parseInt(name, nameAsInt) == 0: nameAsInt = -1
     for no, na in items(winEncodings):
       if no == nameAsInt or eqEncodingNames(na, name): return CodePage(no)
     result = CodePage(-1)
 
-  proc codePageToName*(c: CodePage): string =
+  proc codePageToName(c: CodePage): string =
```

Comment:
Windows specific procs are public. I'm not sure why. I doubt it is worth deprecating it and creating an internal codePageToName2 proc.

Reply:
If you improve the docs, don't un-export procs, it's a breaking change. Please undo.

Reply:
Deferred it.